public_html/register.php

```diff
@@ -1,23 +1,23 @@
 <?php
+session_start();
 require 'connect.php';
-
-$name = $_POST['name'];
-$address = $_POST['address'];
-$username = $_POST['username'];
-$password = $_POST['password'];
-
-
-
-
-
-//$hash = password_hash($password);
-
-
-
-//$sql = "INSERT INTO Users (name, address, username, password, hash)
-//VALUES ('John', 'Doe', 'address')";
-
-
+if($_SERVER["REQUEST_METHOD"] == "POST") {
+  $name = $_POST['name'];
+  $address = $_POST['address'];
+  $username = $_POST['username'];
+  $password = $_POST['password'];
+
+  $hash = password_hash($password);
+
+  $sql = "INSERT INTO Users (name, address, username, password, hash)
+  VALUES ($name, $address, $username, $password, $hash)";
```

Comment:
We need to make this safe. Meaning we need to prevent:
- SQL Injection (by using `prepare-statements`)
- XSS (by using `htmlspecialchars`)

Reply:
yesh